### PR TITLE
add some extra details

### PR DESCRIPTION
**Pull request by Krista-Rogers**
Tested using pipeline - it passed

### CODE REVIEW

File: libnss_ato.c [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/**
```
Comment (tomwatson1024) [inline libnss_ato.c]: One star would match the rest of the file.

File: libnss_ato.c [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+ * Some of the content of this file has been edited by Metaswitch, in the time
+ * period from December 2015 to the present time.
+ * /
```
Comment (tomwatson1024) [inline libnss_ato.c]: ` */` rather than ` * /` would match the rest of the file.

**Review by tomwatson1024 — APPROVED** [review]
Couple of minor style comments. I'm not sure if we care about whitespace changes or not - if we do, there are quite a few in this PR. Personally I think the world is a better place with fewer trailing spaces.

**Comment by Krista-Rogers** [comment]
Fixed up the style changes. I'm not worried about the whitespace changes as they are only in files that are already modified. 🙂

**Pull request merged**